code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm

```diff
@@ -31,10 +31,10 @@
 	X.visible_message(span_xenowarning("\The [X] spews forth a wide cone of acid!"), \
 	span_xenowarning("We spew forth a cone of acid!"), null, 5)
 
-	X.add_movespeed_modifier(type, TRUE, 0, NONE, TRUE, 2)
+	X.add_movespeed_modifier(type, TRUE, 0, NONE, TRUE, 0)
 	start_acid_spray_cone(target, X.xeno_caste.acid_spray_range)
 	add_cooldown()
-	addtimer(CALLBACK(src, .proc/reset_speed), rand(2 SECONDS, 3 SECONDS))
+	addtimer(CALLBACK(src, .proc/reset_speed), rand(0 SECONDS, 0 SECONDS))
```

Comment:
What is going on here? If you're setting everything to 0, just delete it.

Reply:
the code is there just incase it needs to be used in the future

Reply:
Code doesnt work like that. If someone needs to use it in the future, they can add it back in.

Reply:
done